serverless/event_sources/knative-event-sources.adoc

```diff
@@ -5,7 +5,7 @@ include::modules/common-attributes.adoc[]
 :context: knative-event-sources
 toc::[]
 
-An _event source_ is an object that links an event producer with an event _sink_, or consumer. A sink can be a Knative service, channel, or broker that receives events from an event source.
```

Comment:
hm, I'm not totally sure this is correct.  An event source in some cases actually creates the event itself (for example, pingsource) vs 'linking' from elsewhere.  Just so I can understand what you mean, could you elaborate a bit more?

Reply:
@lberk I know what you mean, since the whole point is kind of that things are decoupled, however we say pretty much the same thing about "linking" here upstream: https://knative.dev/docs/eventing/sources/

Do you think perhaps the Eventing WG could crowd source a more accurate description of event sources that we can then apply everywhere it needs to be explained?

Reply:
Could we just use
> that acts as a link between an event producer and an event sink

from upstream directly?  The difference between that wording, and the one proposed in this PR (to me) is the implication of where that linking takes place (external to the source vs either internal or external).  I think the upstream docs is about is accurate as we'll get here without a ton of bikeshedding. (ie, see my comments on this PR :smile:)